From fda64266a8fb9478cef6b37ae2675d59564ee4d0 Mon Sep 17 00:00:00 2001
From: Don <don.j.olmstead@gmail.com>
Date: Tue, 18 Sep 2018 10:18:25 -0700
Subject: [PATCH 1/3] ICU-20136 Check U_HAVE_TZ* values (#114)

* Add check for U_HAVE_TZSET

* Add check for U_HAVE_TZNAME
---
 source/common/putilimp.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/common/putilimp.h b/source/common/putilimp.h
index 8b858df9e3..722bce88be 100644
--- a/source/common/putilimp.h
+++ b/source/common/putilimp.h
@@ -94,7 +94,7 @@ typedef size_t uintptr_t;
 #   define U_NL_LANGINFO_CODESET CODESET
 #endif
 
-#ifdef U_TZSET
+#if defined(U_TZSET) || defined(U_HAVE_TZSET)
     /* Use the predefined value. */
 #elif U_PLATFORM_USES_ONLY_WIN32_API
     // UWP doesn't support tzset or environment variables for tz
@@ -132,7 +132,7 @@ typedef size_t uintptr_t;
 #   define U_TIMEZONE timezone
 #endif
 
-#ifdef U_TZNAME
+#if defined(U_TZNAME) || defined(U_HAVE_TZNAME)
     /* Use the predefined value. */
 #elif U_PLATFORM_USES_ONLY_WIN32_API
     /* not usable on all windows platforms */
-- 
2.19.0.windows.1

